local-setup/jupyter/dataloader/crs_loader.py

```
@@ -499,97 +921,100 @@ def _ensure_stateinfo_for_tenant(self, tenant_code: str, display_name: str = Non
-                 "-v", f"new_data={json_lib.dumps(new_config)}",
-                 "-v", f"mod_code={module_code}",
-                 "-c", update_sql],
-                capture_output=True, text=True, timeout=10
+            import psycopg2
+            
+            conn = psycopg2.connect(
+                host="postgres-db",
+                port="5432", 
+                database="egov",
+                user="egov",
+                password="egov123"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)